### PR TITLE
frontend: improve lighting receive workflow

### PR DESCRIPTION
**Pull request by thisconnect**
This fixes a bug that sometimes showed a qr placeholder instead of
a real code. This happened if the user edited an invoice and changed
amount to invalid amount for example empty amount.

Also added new invoice button on invoice created and success step
and slightly changed the layout to better fit on mobile screen.

### CODE REVIEW

File: frontends/web/src/components/view/view.module.css [hunk 1]
```diff
@@ -119,7 +119,7 @@
 }
 @media (max-width: 768px) {
     .fill {
-        padding: 0 0 var(--space-default) 0;
+        padding: 0 0 var(--space-half) 0;
     }
```
Comment (thisconnect) [inline frontends/web/src/components/view/view.module.css:123]: I'll create a separate PR to master for this.

**Comment by thisconnect** [comment]
CI failing, https://github.com/digitalbitbox/bitbox-wallet-app/commit/5fca9ae1d769f9c9a6b9439b0c45cf7bfda34737#r141714786

**Review by Beerosagos — APPROVED** [review]
tACK!

Pls wait until https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2782 is merged, to avoid having to handle other possible conflicts there  :pray:

**Comment by thisconnect** [comment]
rebased

**Pull request merged**